apps/web/ui/partners/groups/design/application-form/program-application-form.tsx

```
@@ -0,0 +1,319 @@
+          } as z.infer<
+            typeof programApplicationFormLongTextFieldWithValueSchema
+          >;
+        case "select":
+          return {
+            ...field,
+            value: "",
+          } as z.infer<typeof programApplicationFormSelectFieldWithValueSchema>;
+        case "multiple-choice":
+          const multipleChoiceField = field as z.infer<
+            typeof programApplicationFormMultipleChoiceFieldSchema
+          >;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)